### PR TITLE
Add rudimentary playfield

### PR DESCRIPTION
**Pull request by Ryooshuu**
Not finished yet, but usable enough to make a rhythm game with this system. Pull requesting this now so that we have at least a basis that we can work off of. Other features such as a scrolling playfield, object pooling, and other concerns will be PR'd in the future.

Kumi Tests_FAe9luqHtj

### CODE REVIEW

File: Kumi.Game/Gameplay/Clocks/GameplayClockContainer.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Timing;
+
+namespace Kumi.Game.Gameplay.Clocks;
+
+public partial class GameplayClockContainer : Container, IAdjustableClock, IGameplayClock
```
Comment (furiner) [inline Kumi.Game/Gameplay/Clocks/GameplayClockContainer.cs:8]: This class is pretty painfully similar to [osu!lazer's own GameplayClockContainer](https://github.com/ppy/osu/blob/ed02335e3eb802c1865a653eac0af1b1050f3121/osu.Game/Screens/Play/GameplayClockContainer.cs).

Comment (Ryooshuu) [reply Kumi.Game/Gameplay/Clocks/GameplayClockContainer.cs:8]: To be fair, a good majority of the functions here are implementations of the interface, so I think we're fine on that part.

Comment (furiner) [reply Kumi.Game/Gameplay/Clocks/GameplayClockContainer.cs:8]: That would only stand if the function bodies weren't nearly the same as well.

Comment (Ryooshuu) [reply Kumi.Game/Gameplay/Clocks/GameplayClockContainer.cs:8]: Not sure how else you want me to change them, it's the simplest implementation imaginable. One change I could add is just to remove the bindable property of `IsPaused`, but overall everything kinda has to be kept the same.

Comment (furiner) [reply Kumi.Game/Gameplay/Clocks/GameplayClockContainer.cs:8]: Likely better to just add the funny copyright comment block.

Comment (Ryooshuu) [reply Kumi.Game/Gameplay/Clocks/GameplayClockContainer.cs:8]: Probably? I don't see a reason why I should add that if it's something this simple of an implementation. Even then, osu!lazer has a more [complex](https://github.com/ppy/osu/blob/master/osu.Game/Screens/Play/MasterGameplayClockContainer.cs) [implementation](https://github.com/ppy/osu/blob/master/osu.Game/Beatmaps/FramedBeatmapClock.cs) which we don't need to use.

**Review by furiner — APPROVED** [review]
Probably fine to push.

**Pull request merged**